src/plugins/maps_dashboards/public/application.tsx

```diff
@@ -15,12 +15,7 @@ export const renderApp = (
   { appBasePath, element }: AppMountParameters
 ) => {
   ReactDOM.render(
-    <MapsDashboardsApp
-      basename={appBasePath}
-      notifications={notifications}
-      http={http}
-      navigation={navigation}
```

Comment:
navigation seems missing, is it not required?

Reply:
Added it back.